src/project/mod.rs

```diff
@@ -18,6 +18,28 @@ use std::{
 
 use toml_edit::{Array, Document, Item, Table, TomlError, Value};
 
+#[derive(Debug, Copy, Clone)]
+/// What kind of dependency spec do we have
+pub enum SpecType {
+    /// Host dependencies are used that are needed by the host environment when running the project
+    Host,
+    /// Build dependencies are used when we need to build the project, may not be required at runtime
+    Build,
+    /// Regular dependencies that are used when we need to run the project
+    Run,
+}
+
+impl SpecType {
+    /// Convert to a name used in the manifest
+    pub fn to_name(&self) -> &'static str {
```

Comment:
I think a better name would just be `name`.

Reply:
Done